DigitalLearningSolutions.Web/Views/Login/AccountNotApproved.cshtml

```diff
@@ -0,0 +1,12 @@
+﻿@{
+  ViewData["Title"] = "Registration not yet approved";
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <h1 id="page-heading">Registration not yet approved</h1>
```

Comment:
We should be using an appropriate [nhsuk-heading class](https://service-manual.nhs.uk/design-system/styles/typography#headings) on all of the hX elements in the solution, rather than relying on other styling defined ourselves/default styling. Some places are using these, some aren't. We probably need to go back though and make sure they are consistent @stellake

Reply:
Hmm.. I think the h1 etc get the heading class style automatically in the compiled css, but I don't mind deciding on a rule of consistency and following it
![image](https://user-images.githubusercontent.com/40016796/114583180-3f2fb880-9c79-11eb-8bf4-47ec866bc907.png)

Reply:
I agree they probably are getting automatically applied based on that, and it is probably fine. I just think that we should probably be using these components as they are specified in the nhs design system, someone could easily go and change stuff on #page-heading affecting multiple pages.
![image](https://user-images.githubusercontent.com/59561751/114584596-bfa2e900-9c7a-11eb-82d3-5bc6c13c089b.png)

Reply:
Oh I see what you mean - you think we should replace the id="page-heading" with the NHS class, but I thought the initial comment was about whether we should be explicit about the NHS style or whether being implicit is ok.

**The explicit vs implicit** - I think my preference would be explicit, which means that I'm happy to set the standard as "use the typography classes explicitly".

**The use of "page-heading" vs NHS typography classes** - I don't think it should be X or Y as the id is specified on some of the pages to identify the main header (e.g. it's used in full screen learning menu etc) and to specify some styling in addition to the NHS styling, which is something HEE have asked us to do. I'd prefer the header styling to be consistent across the site, so I don't think we should remove the id, but if we go for explicit styles, we should add the class in addition to the id.